Changelog.md

```diff
@@ -18,6 +18,7 @@
 ## Features
 * Add links to the aspects and followed tags pages on mobile [#7265](https://github.com/diaspora/diaspora/pull/7265)
 * diaspora\* is now available in Gàidhlig, Occitan, and Schwiizerdütsch
+* admins cannot be blocked anymore [#7279](https://github.com/diaspora/diaspora/issues/7297)
```

Comment:
Ok, just drop the PR. It was a try. I realize that this is the wrong way to implement this. I see you can still unshare the admin.